orchestration/templates/ingest-hca.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: ingest-hca
+spec:
+  entrypoint: main
+  serviceAccountName: {{ .Values.serviceAccount.k8sName }}
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: source-bucket-name
+          - name: source-bucket-prefix
```

Comment:
Sorry, one last thing: I think it'd be good to include an `output-prefix` as an arg here, and pass it through to the dataflow job. That way we can reprocess the same inputs multiple times without the outputs colliding.